modules/ocf_decal/manifests/init.pp

```diff
@@ -0,0 +1,21 @@
+class ocf_decal {
+
+  include ocf_decal::website
+
+  class { 'apache':
+    default_vhost => false;
+  }
+
+  user { 'ocfdecal':
```

Comment:
Why not use the existing `decal` user?

Reply:
I figured it was marginally better/safer to create a new user for this purpose.

Reply:
Are we still going to be using the existing decal user then

Reply:
I'm not sure, but I think not.

Reply:
Also, if you create users in puppet like this, you'll want to designate them as system users lest they collide with LDAP users

Reply:
(Or alternatively fix the UID to something guaranteed not to be used in the future)

Reply:
Agreed about making it a system user instead, we've had problems with the user ids colliding in the past.

Reply:
this is done

Reply:
I manually changed the uid and gid on the VM